docs/DeveloperGuide.md

```diff
@@ -290,31 +290,34 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC1 - List all people saved in the System**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to list all people saved in the System by entering a command.
+2.  System shows a list of persons.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. If there are no contacts stored in the system.
 
-  Use case ends.
+  * 1a1 System informs user that they have no contacts stored.
+  
+    * Use case ends.
 
-* 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+* 1b. The System detects an error in the entered command.
 
-      Use case resumes at step 2.
+    * 1b1 System requests for a valid command.
+
+      Use case resumes at step 1.
```

Comment:
Same as the add contact use case PR, could you remove the indentation to align with Kenneth's format?